.goreleaser.yml

```diff
@@ -14,12 +14,13 @@ builds:
   ldflags:
   - -s -w -X github.com/sachaos/toggl.version={{.Version}}
 archives:
-- replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
+- name_template: >-
+    {{- .ProjectName }}_
+    {{- .Version }}_
+    {{- title .Os }}_
+    {{- if eq .Arch "amd64" }}x86_64
+    {{- else if eq .Arch "386" }}i386
+    {{- else }}{{ .Arch }}{{ end -}}
```

Comment:
The name_template keeps the naming format.

https://github.com/sachaos/toggl/releases/tag/v0.5.1